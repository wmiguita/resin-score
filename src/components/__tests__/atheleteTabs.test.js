import React from "react"
import renderer from "react-test-renderer"

import AtheleteTabs from "../atheleteTabs"


describe( "AtheleteTabs", () => {
  it( "should render athele tab options", () => {
    const tree = renderer.create( <AtheleteTabs selected={ 1 } onChange={ () => {} } /> ).toJSON()

    expect( tree ).toMatchSnapshot()
  })
})
