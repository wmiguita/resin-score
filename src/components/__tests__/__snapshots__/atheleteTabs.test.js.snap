// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AtheleteTabs should render athele tab options 1`] = `
<div
  className="MuiTabs-root"
>
  <div
    className="MuiTabs-scroller MuiTabs-fixed"
    onScroll={[Function]}
    style={
      Object {
        "marginBottom": null,
        "overflow": "hidden",
      }
    }
  >
    <div
      aria-label="Athelete menu"
      className="MuiTabs-flexContainer"
      onKeyDown={[Function]}
      role="tablist"
    >
      <button
        aria-controls="atheletes-list"
        aria-selected={false}
        className="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit"
        disabled={false}
        id="atheletes-list"
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="tab"
        tabIndex={-1}
        type="button"
      >
        <span
          className="MuiTab-wrapper"
        >
          Atheletes
        </span>
      </button>
      <button
        aria-controls="manage-athelete"
        aria-selected={true}
        className="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit Mui-selected"
        disabled={false}
        id="manage"
        onBlur={[Function]}
        onClick={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="tab"
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiTab-wrapper"
        >
          Manage
        </span>
        <span
          className="PrivateTabIndicator-root-1 PrivateTabIndicator-colorSecondary-3 MuiTabs-indicator"
          style={Object {}}
        />
      </button>
    </div>
  </div>
</div>
`;
